src/admin/content-block/components/wrappers/MediaPlayerWrapper/MediaPlayerWrapper.tsx

```diff
@@ -43,14 +59,16 @@ export const MediaPlayerWrapper: FC<MediaPlayerProps> = ({ item, title, width })
 				'u-center-m'
 			)}
 		>
-			<BlockFlowPlayer
-				title={title}
-				src={playerTicket ? playerTicket.toString() : null}
-				poster="https://via.placeholder.com/1920x1080"
-				onInit={get(item, 'value') ? initFlowPlayer : () => {}}
-				token={getEnv('FLOW_PLAYER_TOKEN')}
-				dataPlayerId={getEnv('FLOW_PLAYER_ID')}
-			/>
+			{!!videoStill && (
+				<BlockFlowPlayer
+					title={title}
+					src={playerTicket ? playerTicket.toString() : null}
+					poster={videoStill}
+					onInit={get(item, 'value') ? initFlowPlayer : () => {}}
+					token={getEnv('FLOW_PLAYER_TOKEN')}
+					dataPlayerId={getEnv('FLOW_PLAYER_ID')}
+				/>
+			)}
```

Comment:
als er geen video still wordt gevonden, dan renderen we de video speler niet? Dat lijkt me wat extreem. Mss een status bijhouden voor het laden van de video still, als state === 'loading', dan toon je nog geen video speler. Als de state === 'loaded' || state === 'error' dan toon je de video speler met of zonder poster.

Reply:
Ben dit in de PR aan het oplossen waar de media-player ook niet herlaadt bij het selecteren van een andere item.